Python/ReviewSession10/EisAutoAcceptance/matching.py

```diff
@@ -0,0 +1,384 @@
+import codecs
+import datetime as dt
+import re
+import time
+from configparser import ConfigParser
+
+from bs4 import BeautifulSoup as bs
+from peewee import fn
+from progress.bar import IncrementalBar
+from selenium.common.exceptions import (ElementClickInterceptedException,
+                                        StaleElementReferenceException,
+                                        TimeoutException, WebDriverException)
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.support.wait import WebDriverWait
```

Comment:
I see you are using this a lot and in most even line by line so maybe shorten it so it would be cleaner like

`from selenium.webdriver.support.wait import WebDriverWait as wdr
`